src/core/abr/network_analyzer.ts

```diff
@@ -291,8 +339,8 @@ export default class NetworkAnalyzer {
       }
     }
 
-    if (clockTick.speed > 1) {
-      newBitrateCeil /= clockTick.speed;
+    if (playbackInfo.speed > 1) {
+      newBitrateCeil /= playbackInfo.speed;
```

Comment:
Would it be desirable to define a newBitrateCeil even if the speed if inferior to 1 ?
Maybe we could download better content when speed is low.